examples/CMakeLists.txt

```diff
@@ -22,6 +21,7 @@ if(NOT ${_HEADER_ONLY} OR NOT ${USE_CUDA})
   add_subdirectory(custom_format)
   add_subdirectory(custom_converter)
   add_subdirectory(custom_experiment)
+  add_subdirectory(example_experiment)
```

Comment:
I couldn't make this compile when `USE_CUDA=OFF`. Probably due to the file extension.